modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -906,16 +906,11 @@ protected function add_attributes_to_container( $container, $item_id ) {
 	}
 
 	protected function get_initial_config(): array {
-		if ( Plugin::$instance->experiments->is_feature_active( 'e_nested_atomic_repeaters' ) ) {
```

Comment:
Only remove the if statement. Line 909 and 916. Bring back `support_improved_repeaters`.